CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [3.2.4]
```

Comment:
```suggestion
## [3.2.5]
```

This should ine up with the package.json version..and hopefully the tagged built version.

Reply:
Whoops; good catch